core/state_processor_rip7560.go

```diff
@@ -632,7 +643,12 @@ func validatePaymasterReturnData(data []byte) ([]byte, uint64, uint64, error) {
 		return nil, 0, 0, errors.New("paymaster returned context size too large")
 	}
 	magicExpected := common.Bytes2Hex(validatePaymasterResult.ValidationData[:20])
-	if magicExpected != common.Bytes2Hex(MAGIC_VALUE_PAYMASTER[:]) {
+	if magicExpected == common.Bytes2Hex(MAGIC_VALUE_PAYMASTER[:]) {
+	} else if magicExpected == common.Bytes2Hex(MAGIC_VALUE_SIGFAIL[:]) {
+		if !estimate {
+			return nil, 0, 0, errors.New("paymaster return MAGIC_VALUE_SIGFAIL")
+		}
+	} else {
```

Comment:
Same as above, could you change the sequence? Or is it the code from the infinitism github?

Reply:
Infinitism style applied